language/de/migrator.php

```diff
@@ -61,6 +61,7 @@
 	'MIGRATION_REVERT_DATA_IN_PROGRESS'	=> 'Mache Daten-Änderungen rückgängig: %1$s; Dauer: %2$.2f Sekunden',
 	'MIGRATION_REVERT_DATA_RUNNING'		=> 'Mache Daten-Änderungen rückgängig: %s.',
 	'MIGRATION_REVERT_SCHEMA_DONE'		=> 'Rückgängig gemachte Schema-Änderungen: %1$s; Dauer: %2$.2f Sekunden',
+	'MIGRATION_REVERT_SCHEMA_IN_PROGRESS'	=> 'Rückgängig machen von Schema-Änderungen: %1$s; Dauer: %2$.2f Sekunden',
 	'MIGRATION_REVERT_SCHEMA_RUNNING'	=> 'Mache Schema-Änderungen rückgängig: %s.',
```

Comment:
Das passt nicht zu den bisherigen Ausdrücken: In meinen Augen müsste das 
> Mache Schema-Änderungen rückgängig: ....

heißen.